docs/tools/automated-testing/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Automated Testing
+layout: en
+permalink: docs/tools/automated-testing/overview/
+---
+
+Running an automated test for your mobile apps – native, hybrid or web Application is better than a manual test in some cases, since it doesn't rely on human resources in order to be run, hence it is cheaper, more relaiable and more accurate than the manual testing.
```

Comment:
Also, description needs to be more generic: just write what automated testing is, what kind of automated testing you can do on testobject (functional), mention continuous delivery and continuous testing (read up on these if needed).